ytdownloader/converter.py

```
@@ -0,0 +1,103 @@
+import discord
+from redbot.core import commands, Config
+from yt_dlp import YoutubeDL
+import asyncio
+from redbot.core import data_manager
+from pathlib import Path
```

Ruff (0.8.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)